pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -879,7 +892,17 @@ func (oe *operationExecutor) generateUnmountDeviceFunc(
 				deviceToDetach.VolumeSpec.Name(),
 				err)
 		}
-
+		refs, err := attachableVolumePlugin.GetDeviceMountRefs(deviceMountPath)
+		if err != nil || len(refs) > 0 {
+			if err == nil {
+				err = errors.New(fmt.Sprintf("The device mount path %q are still mounted by other references %v", deviceMountPath, refs))
```

Comment:
Instead of `errors.New(fmt.Sprintf(...))` you can do `fmt.Errorf(...)`. Also, nit, `are still mounted` -> `is still mounted`.